imagekitio/models/results/EmbeddedMetadata.py

```diff
@@ -1,6 +1,6 @@
 class EmbeddedMetadata:
-    def __init__(self, x_resolution=None, y_resolution=None, date_created=None, date_timecreated=None):
-        self.x_resolution = x_resolution
-        self.y_resolution = y_resolution
-        self.date_created = date_created
-        self.date_timecreated = date_timecreated
+    def __init__(self, XResolution=None, YResolution=None, DateCreated=None, DateTimeCreated=None):
+        self.XResolution = XResolution
```

Comment:
Data structure for embedded metadata is this - https://docs.imagekit.io/api-reference/media-api#embedded-metadata-object-structure
What will SDK do if other values are there?